apps/web/src/features/portfolio/Portfolio.module.css

```
@@ -2,31 +2,25 @@
 .portfolioSection {
-  --portfolio-primary: #9333ea; /* Purple - Phoenix VC brand */
-  --portfolio-accent: #f59e0b; /* Amber/Orange - accent */
-  --portfolio-gradient-start: #9333ea;
-  --portfolio-gradient-end: #f59e0b;
+  --portfolio-primary: hsl(
+    var(--color-primary)
+  ); /* Purple - Phoenix VC brand */
+  --portfolio-accent: hsl(var(--color-accent)); /* Amber/Orange - accent */
+  --portfolio-gradient-start: hsl(var(--color-primary));
+  --portfolio-gradient-end: hsl(var(--color-accent));
   min-height: 100vh;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)